### PR TITLE
fix: escaped property name

### PR DESCRIPTION
**Pull request by w-vi**

### CODE REVIEW

File: test/test-Signature.cc [hunk 1]
```diff
@@ -138,7 +138,7 @@ TEST_CASE("Identifier enclosed by backticks", "[signature]")
     REQUIRE(blueprint.report.error.code == Error::OK);
     REQUIRE(blueprint.report.warnings.empty());
 
-    REQUIRE(signature.identifier == "username `is` g``ood");
+    REQUIRE(signature.identifier == "```username `is` g``ood```");
```
Comment (pksunkara) [inline test/test-Signature.cc]: Can you confirm that the property name is this case will be 

```
username `is` g``ood
```

Comment (w-vi) [inline test/test-Signature.cc]: So then lets figure out how to achieve it.

**Comment by pksunkara** [comment]
Only a single question. Otherwise, lgtm.

File: test/test-Signature.cc (hunk 1 above)
Comment (w-vi) [inline test/test-Signature.cc]: let me check one more time

Comment (pksunkara) [inline test/test-Signature.cc]: I think it is before the PR for sure, not sure how it changed after this PR.

Comment (w-vi) [inline test/test-Signature.cc]:
```apib
FORMAT: 1A

# API

## A [GET /]

+ Response 200 (application/json)

    + Attributes (object)
        + ```username `is` g``ood```: username
```
result 

```
{
  "element": "parseResult",
  "content": [
    {
      "element": "category",
      "meta": {
        "classes": [
          "api"
        ],
        "title": "API"
      },
      "attributes": {
        "meta": [
          {
            "element": "member",
            "meta": {
              "classes": [
                "user"
              ]
            },
            "content": {
              "key": {
                "element": "string",
                "content": "FORMAT"
              },
              "value": {
                "element": "string",
                "content": "1A"
              }
            }
          }
        ]
      },
      "content": [
        {
          "element": "category",
          "meta": {
            "classes": [
              "resourceGroup"
            ],
            "title": ""
          },
          "content": [
            {
              "element": "resource",
              "meta": {
                "title": "A"
              },
              "attributes": {
                "href": "/"
              },
              "content": [
                {
                  "element": "transition",
                  "meta": {
                    "title": "A"
                  },
                  "attributes": {
                    "href": "/"
                  },
                  "content": [
                    {
                      "element": "httpTransaction",
                      "content": [
                        {
                          "element": "httpRequest",
                          "attributes": {
                            "method": "GET"
                          },
                          "content": []
                        },
                        {
                          "element": "httpResponse",
                          "attributes": {
                            "statusCode": "200",
                            "headers": {
                              "element": "httpHeaders",
                              "content": [
                                {
                                  "element": "member",
                                  "content": {
                                    "key": {
                                      "element": "string",
                                      "content": "Content-Type"
                                    },
                                    "value": {
                                      "element": "string",
                                      "content": "application/json"
                                    }
                                  }
                                }
                              ]
                            }
                          },
                          "content": [
                            {
                              "element": "dataStructure",
                              "content": [
                                {
                                  "element": "object",
                                  "content": [
                                    {
                                      "element": "member",
                                      "content": {
                                        "key": {
                                          "element": "string",
                                          "content": "username `is` g``ood"
                                        },
                                        "value": {
                                          "element": "string",
                                          "content": "username"
                                        }
                                      }
                                    }
                                  ]
                                }
                              ]
                            },
                            {
                              "element": "asset",
                              "meta": {
                                "classes": [
                                  "messageBody"
                                ]
                              },
                              "attributes": {
                                "contentType": "application/json"
                              },
                              "content": "{\n  \"username `is` g``ood\": \"username\"\n}"
                            },
                            {
                              "element": "asset",
                              "meta": {
                                "classes": [
                                  "messageBodySchema"
                                ]
                              },
                              "attributes": {
                                "contentType": "application/schema+json"
                              },
                              "content": "{\n  \"$schema\": \"http://json-schema.org/draft-04/schema#\",\n  \"type\": \"object\",\n  \"properties\": {\n    \"username `is` g``ood\": {\n      \"type\": \"string\"\n    }\n  }\n}"
                            }
                          ]
                        }
                      ]
                    }
                  ]
                }
              ]
            }
          ]
        }
      ]
    }
  ]
}
```

Seems OK

**Comment by pksunkara** [comment]
Merge away when tests pass.

**Comment by w-vi** [comment]
Appveyor status update seems not to be working ... see https://ci.appveyor.com/project/Apiary/snowcrash/history

**Pull request merged**